spec/requests/api/v1/users_spec.rb

```diff
@@ -133,7 +133,7 @@
 
     context "when unauthenticated" do
       it "returns unauthorized" do
-        put api_user_suspend_path(id: target_user.id),
+        put api_user_add_role_path(id: target_user.id, role: "suspended"),
```

Comment:
Ditto here, since this is no longer being implemented in the users controller I think these specs should be moved to/consolidated with the ones in `user_roles_spec`